grunt/tasks/javascript.js

```diff
@@ -157,6 +157,16 @@ module.exports = function(grunt) {
       });
     }
 
+    // Collect react templates
+    const reactTemplatePaths = [];
+    options.reactTemplates.forEach(pattern => {
+      grunt.file.expand({
+        filter: options.pluginsFilter
```

Comment:
@brian-learningpool It already does the filter identical to the way the plugins currently work

Reply:
Awesome.  👍